docs/admin/access.rst

```diff
@@ -193,15 +212,15 @@ To manage permissions for a whole instance at once, add users to
 appropriate :ref:`default groups <default-groups>`:
 
 * `Users` (this is done by default by the
-  :ref:`automatic group assignment <autogroup>`)
+  :ref:`automatic group assignment <autogroup>`).
 * `Reviewers` (if you are using :ref:`review workflow <reviews>` with dedicated
-  reviewers)
-* `Managers` (if you want to delegate most of management operations to somebody
+  reviewers).
+* `Managers` (if you want to delegate most of the management operations to somebody
   else).
 
 You should keep all projects configured as `Public` (see :ref:`acl`), otherwise
 the site-wide permissions provided by membership in `Users` and `Reviewers` groups
```

Comment:
the `Users` and `Reviewers`

Reply:
Yes, can you add a suggestion for `the`?